### PR TITLE
fix: rename menu label

### PR DESCRIPTION
**Pull request by bjp232004**
issue #4872

## Summary by CodeRabbit

- **New Features**
	- Updated menu label from "Ingestion" to "Data sources" for improved clarity in the user interface.
	- Expanded localization support with numerous new terms and phrases for English, German, Spanish, French, Italian, Japanese, Korean, Dutch, Portuguese, Turkish, and Chinese, enhancing user experience across multiple sections including menus, dashboards, and search functionalities.

### CODE REVIEW

**Review by hengfeiyang — APPROVED** [review]

File: web/src/locales/languages/en.json [hunk 1]
```diff
@@ -145,7 +145,7 @@
     "tickets": "Ticket",
     "organizations": "Organizations",
     "function": "Functions",
-    "ingestion": "Ingestion",
+    "ingestion": "Data sources",
```
[the remark this answers is not in the document]
Comment (bjp232004) [reply web/src/locales/languages/en.json:148]: both menu label and page title changed to Data sources

File: web/src/locales/languages/ja.json [hunk 2]
```diff
@@ -686,6 +845,100 @@
...
+    "timeRange": "時間範囲",
+    "frequency": "周波数",
+    "lastTriggeredAt": "最終トリガー日時",
+    "createdAt": "作成日",
+    "tab": "タブ",
+    "cached": "キャッシュ",
+    "cachedReport": "キャッシュレポート",
+    "destination": "目的地",
+    "scheduled": "スケジュール済み"
+  },
```
[the remark this answers is not in the document]
Comment (bjp232004) [reply web/src/locales/languages/ja.json:925]: Need help to review this

File: web/src/locales/languages/es.json [hunk 3]
```diff
@@ -102,7 +155,10 @@
     "docs": "Documentos",
     "billings": "Facturación",
     "rum": "Monitorización de usuarios reales",
-    "settings": "Ajustes"
+    "settings": "Ajustes",
+    "iam": "SOY YO",
+    "report": "Informes",
+    "pipeline": "Tuberías"
```
[the remark this answers is not in the document]
Comment (bjp232004) [reply web/src/locales/languages/es.json:161]: Considered and fixed

File: web/src/locales/languages/de.json [hunk 4]
```diff
@@ -686,6 +845,100 @@
...
+    "queryRange": "Abfragebereich",
+    "refreshQuery": "Abfrage aktualisieren",
+    "queryType": "Art der Abfrage"
+  },
+  "traces": {
+    "viewLogs": "Protokolle anzeigen",
+    "backToLogs": "Zurück zu Logs"
   }
-}
+}
```
[the remark this answers is not in the document]
Comment (bjp232004) [reply web/src/locales/languages/de.json:944]: Checked with google translation and it give "pipeline".

**Pull request merged**